lib/ansible/cli/doc.py

```diff
@@ -511,11 +511,16 @@ def _dump_yaml(struct, indent):
 
     @staticmethod
     def add_fields(text, fields, limit, opt_indent, return_values=False, base_indent=''):
+        customized_output_fields = (
```

Comment:
It's probably better to move this constant (and the other similar constant for top-level processing) to a proper place so the lists aren't regenerated on every function calls. (And it's probably better to make them a `set`/`frozenset`.)

Reply:
I kept constant definitions close to the code using them in order to avoid adding yet another global variable (even if they would be called class variables in this particular case). But I can move them somewhere else.

As for the why I used tuple: I just used what the rest of the code already uses. I am not sure if the performance of the set would be any better in this case where we only have a few items in a list.